### DISCUSSION TITLE
Only certain object in  blend file are visible.

### DISCUSSION THREAD

**Discussion by rohan-nara** [google/filament#5780] [Q&A]
I am trying to view my 3d model in mobile using android studio filament library 
After exporting my .blend file to .glb in Blender
The issue I faced was only certain parts of the mesh are visible , that is only the eyes and mouth of the character are visible in the modelviewer loadgltf , not the whole character mesh don't know why this happen.

this is the code snippet:

      private fun loadGlb(name: String) {
        Log.v("working","glb function called")
        val buffer = readAsset("models/${name}.glb")
        modelViewer.loadModelGlb(buffer)
        modelViewer.transformToUnitCube()

snippet by: https://medium.com/@philiprideout/getting-started-with-filament-on-android-d10b16f0ec67
![png](https://user-images.githubusercontent.com/64644909/178506174-4fcaafba-08c0-4bd3-9396-41631dc5b138.png)
only this part  of the character is visible 

I have checked the face orientation  of the 3d meshes and the normal maps.

**Comment by bejado** [comment]
Do you have any lights in your scene (such as an IndirectLight or Sun)? It's possible the eyes and mouth have emission, which is why they don't require lighting.

**Reply by rohan-nara** [reply]
Yes , Have a HDR environment  setup which is working fine

**Reply by rohan-nara** [reply]
The exported  Gltf file (.gltf + .bin)  from blender(Checked on 2.83 , 2.92 , 3.1) versions  are not  loading  into the  model viewer
The exported  Gllb file  from blender(Checked on 2.83 , 2.92 , 3.1) versions  are  loading into the model viewer , but entire mesh is not visible, only part of the mesh is visible ,  and animation  is playing  .
Also please let me know the exact  glb , gltf format supported by filament.

**Reply by bejado** [reply]
We support these glTF 2.0 features: https://github.com/google/filament#gltf-20

Are you able to upload your glTF model here?

**Reply by rohan-nara** [reply]
[arya-1.zip](https://github.com/google/filament/files/9108077/arya-1.zip)

This is embedded glTF model. it has no .bin file in it .


 
[arya-1glb.zip](https://github.com/google/filament/files/9108098/arya-1glb.zip)

This is glb version of the same model

**Reply by bejado** [reply]
Thanks for uploading it. Your model has more bones than Filament supports. When I try to load that model in our desktop glTF viewer, I get the following error:

```
utils::PreconditionPanic
in RenderableManager::Builder::Result filament::RenderableManager::Builder::build(filament::Engine &, utils::Entity):235
in file filament/src/components/RenderableManager.cpp
reason: bone count > 256
```

Inspecting the model with Blender shows it has over 700 bones.
